doc/JSON_FLAGS.md

```diff
@@ -1094,9 +1094,16 @@ Those flags are added by the game code to specific items (that specific welder,
 
 - ```SCEN_ONLY``` Profession can be chosen only as part of the appropriate scenario.
 
-- ```ALLOW_OUTSIDE``` Allows placing player outside of building, useful for outdoor start
-- ```BOARDED``` Start in boarded building (windows and doors are boarded, movable furniture is moved to windows and doors)
-- ```SUR_START``` ... surrounded start, zombies outside the starting shelter.
+- ```ALLOW_OUTSIDE``` Allows placing player outside of building, useful for outdoor start.
+- ```BAD_DAY``` Player starts the game drunk, depressed and sick with the flu.
+- ```BOARDED``` Start in boarded building (windows and doors are boarded, movable furniture is moved to windows and doors).
+- ```CHALLENGE``` Game won't choose this scenario in random game types.
+- ```CITY_START``` Scenario is available only when city size value in world options is more than 0.
+- ```FIRE_START``` Player starts the game with fire nearby.
+- ```HELI_CRASH``` Player starts the game with various limbs wounds.
```

Comment:
Also inside or nearby a crashed helicopter.

Reply:
No, it's not. `"map_special": "mx_helicopter"` is responsible for spawn point. I used `HELI_CRASH` flag in my own scenarios to spawn in wilderness with limb wounds and no helicopters whatsoever.

Reply:
Yes, my bad.